### PR TITLE
Formatting changes

### PR DESCRIPTION
**Pull request by snim2**
rustfmt has changed recently, and this PR brings the repo up to date...

### CODE REVIEW

**Comment by snim2** [comment]
This has failed because Travis needs to update its version of rustfmt, which it caches. In the branch, I did this by pushing a change to the `.travis.yml` to force-update rustfmt, then I dropped that commit. I guess the same trick is needed here to fix master :(

File: src/lib/emitters.rs [hunk 1]
```diff
@@ -90,7 +90,7 @@ pub fn write_json_to_stream<T: RenderJson, U: RenderJson>(mut stream: Box<Write>
         .write_all(format!("{{\n{},\n{}\n}}\n",
                            store.render_json(4),
                            script.render_json(4))
-                           .as_bytes())
+                       .as_bytes())
```
Comment (ltratt) [inline src/lib/emitters.rs]: Is this right? The new formatting makes no sense.

File: src/lib/matchers.rs [hunk 2]
```diff
@@ -678,9 +677,9 @@ mod tests {
                             "{\n\"src\": 1,\n\"dest\": 3\n}",
                             "{\n\"src\": 0,\n\"dest\": 0\n}",
                             "{\n\"src\": 2,\n\"dest\": 4\n}"]
-                .iter()
-                .map(|s| String::from(*s))
-                .collect::<Vec<String>>();
+            .iter()
```
Comment (ltratt) [inline src/lib/matchers.rs]: This indenting makes little sense either.

**Comment by ltratt** [comment]
I wonder if we need to tweak some sort of setting: in a couple of cases rustfmt has really made the indentation more confusing IMHO.

File: src/lib/matchers.rs (hunk 2 above)
Comment (snim2) [inline src/lib/matchers.rs]: It isn't easy to see from the context of the diff, which I suspect is wrong (the diff I see in gitk is not the same as the one here on GH). This is the new formatting:
```rust
        let expected = vec!["\"matches\": [",
                            "{\n\"src\": 1,\n\"dest\": 3\n}",
                            "{\n\"src\": 0,\n\"dest\": 0\n}",
                            "{\n\"src\": 2,\n\"dest\": 4\n}"]
            .iter()
            .map(|s| String::from(*s))
            .collect::<Vec<String>>();
```
This is the same text from master branch:
```rust
        let expected = vec!["\"matches\": [",
                            "{\n\"src\": 1,\n\"dest\": 3\n}",
                            "{\n\"src\": 0,\n\"dest\": 0\n}",
                            "{\n\"src\": 2,\n\"dest\": 4\n}"]
                .iter()
                .map(|s| String::from(*s))
                .collect::<Vec<String>>();
```

Comment (ltratt) [inline src/lib/matchers.rs]: Admittedly, neither indentation level makes any sense to me whatsoever... If we're going to use rustfmt vanilla, we might need to split this into two logical lines, as at the moment it's just too confusing for a human to read IMHO.

**Comment by snim2** [comment]
Needs squashing

**Comment by ltratt** [comment]
OK, although I still don't like the indentation, I guess I can live with it. Squash away!

**Comment by snim2** [comment]
Splat.

**Pull request merged**